pandas/core/groupby/groupby.py

```diff
@@ -487,10 +496,10 @@ def __init__(
         obj: FrameOrSeries,
         keys: Optional[_KeysArgType] = None,
         axis: int = 0,
-        level=None,
+        level: Optional[IndexLabel] = None,
```

Comment:
I *think* IndexLabel is too permissive here no? Would this ever just be a scalar?